src/wasm2asm.h

```diff
@@ -394,7 +389,8 @@ Ref Wasm2AsmBuilder::processFunction(Function* func) {
   for (auto f : frees[f64]) {
     ValueBuilder::appendToVar(theVar, f, makeAsmCoercedZero(ASM_DOUBLE));
   }
-  // if (theVar[1]->size() == 0) { XXX ???
+  // if (theVar[1]->size() == 0) {
+  //   XXX What should splice do???
```

Comment:
Ah, good reword of the comment here to be more explicit about what's "???"

Yeah this seems like the kind of thing we should know? But also seems like the kind of thing that should already be broken. Which ties in nicely to the whole concept of "having tests."

Which is to say I'm ok with leaving this as-is until we have a failing test to motivate what it should be. I also think we should get some tests for  pretty much asap.